File: frontend/src/models/Container.ts
export default class Container {
    container_id!: string;
    name!: string;
    ip_address!: string;
    ip6_address!: string;
    image!: string;
    state!: string;

    constructor(jsonObj?: Container) {
        if (jsonObj) {
            this.container_id = jsonObj.container_id;
            this.name = jsonObj.name;
            this.ip_address = jsonObj.ip_address;
            this.ip6_address = jsonObj.ip6_address;
            this.image = jsonObj.image;
            this.state = jsonObj.state;
        }
    }
}

File: backend/apps/container/container.py
class Container:
    def __init__(self, container_id, name, ip_address, ip6_address, image, state):
        self.container_id = container_id
        self.name = name
        self.ip_address = ip_address
        self.ip6_address = ip6_address
        self.image = image
        self.state = state